README.md

```diff
@@ -144,12 +139,26 @@ When keys are generated, you can configure your DNS server by just pasting the c
 
 #### Custom user changes & patches
 
-If you'd like to change, patch or alter files or behavior of `docker-mailserver`, you can use a script. See the [documentation](https://docker-mailserver.github.io/docker-mailserver/edge/config/advanced/user-patches/) for a detailed explanation.
+If you'd like to change, patch or alter files or behavior of `docker-mailserver`, you can use a script. See the [documentation](https://docker-mailserver.github.io/docker-mailserver/edge/config/advanced/override-defaults/user-patches/) for a detailed explanation.
+
+#### Updating `docker-mailserver`
+
+``` BASH
+docker-compose pull
+docker-compose down
+docker-compose up -d mailserver
+```
+
+You're done! And don't forget to have a look at the remaining functions of the `setup.sh` script with `./setup.sh help`.
 
 #### Supported Operating Systems
 
 We are currently providing support for Linux. Windows is _not_ supported and is known to cause problems. Similarly, macOS is _not officially_ supported - but you may get it to work there. In the end, Linux should be your preferred operating system for this image, especially when using this mailserver in production.
 
+#### Bare Domains
+
+If you want to use a bare domain (host name = domain name), see [FAQ](https://docker-mailserver.github.io/docker-mailserver/edge/faq#can-i-use-nakedbare-domains-no-host-name).
```

Comment:
That could cause confusion. Someone may be tempted to fill both fields with the same string now.

Reply:
I don't see how that differs from `host name = domain name` tbh. You're still implying one to equate to the other. The suggestion I've provided references a "bare domain" as `hostname` being equivalent to `domainname`, it still requires you to read the FAQ if you want that to happen which is what this line is all about directing the user to.

Making it clear that there is a relation to `hostname` and `domainname` fields as they are referenced elsewhere in the document and the `docker-compose.yml` configs only improves the discovery. If you feel there is risk for confusion, add some additional context to clarify:

```
bare domain (where `hostname` would be equivalent to `domainname`)
bare domain (eg `hostname` matching `domainname`)
bare domain (eg `example.com` instead of `mail.example.com`)
bare domain (without a subdomain aka `hostname`)
```

Or something along those lines? I might not be understanding the description well enough myself if I've described that wrong above.